src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -23,12 +23,12 @@ internal class NTAuthentication
         private string _clientSpecifiedSpn;
 
         private int _tokenSize;
-        private Interop.SspiCli.ContextFlags _requestedContextFlags;
-        private Interop.SspiCli.ContextFlags _contextFlags;
+        private ContextFlagsPal _requestedContextFlags;
+        private ContextFlagsPal _contextFlags;
 
         private bool _isCompleted;
         private string _protocolName;
-        private SecSizes _sizes;
+        private object _sizes;
```

Comment:
Forgot to explain why this was changed. @bartonjs, SecSizes is used only in the Windows code and the type is not defined for Unix. So instead of a dummy type, we used object. @shrutigarg will look into possible refactoring of this.